config.yml

```
@@ -1,16 +1,19 @@
 default:
-  data_prep_inputs_path: "/inputs"
   data_prep_outputs_path: "/outputs"
+  asset_impact_data_path: "/inputs"
+  factset_data_path: ""
```

```suggestion
  factset_data_path: "/inputs"
```

if this is `""`, then the script would expect to find those files at the current working directory (which is unlikely). defaulting to `/inputs` mirrors the current behavior of "throw everything into a single dir, and mount it to `/inputs`."

---

so the strategy with the config file is:
- list every possible parameter in the default config, but if it's a parameter that does not have a logical default value, e.g. one that necessarily is specific to a given quarter, then leave the default value null/blank/whatever. The intention there is that if one has not overrided the default value in their selected config, then it should fail, because there's no way this should work properly unless that has been explicitly set
- in each of the quarter specific configs, strive to minimize the amount of parameters set by inheriting as much as possible from the default config, but for parameters that are necessarily specific to that quarter/config, explicitly set them here in GH so that if one day someone comes back to question "which version of FactSet is supposed to be used to build 2022Q4 data?" it will be clearly recorded here in the config. (Though, if one manually edited that before running data.prep, the setting that they used would be captured in the manifest file... so if one was working backwards in the other direction, e.g. "what version of the FactSet data was used to build the PACTA data in this folder?", they could determine that by looking at the manifest in that folder)

With the last few commits I made here, the `factset_data_path` is required for any given quarter. That's the only way I see at the moment to have it clearly known precisely which version of the FactSet data was/should be used.

---

For further explanation.... I'd prefer that the filenames of the FactSet data files have a/the timestamp integrated into them, but in lieu of that the directory name from which they come is the only place where that is currently defined.

---

That makes sense. I've opened https://github.com/RMI-PACTA/workflow.factset/issues/16, which can make good use of the changes in this PR.